### ISSUE TITLE
Guid.NewGuid() throws Exception

### ISSUE THREAD

**Issue by Skleni** [Elringus/DotNetJS#2]
Hey, thank you for this awesome project! I'm currently trying to get some existing .NET code to run in a VS Code web extension, so it came just at the right time for me, too.

I got the basics to work, however a call to `Guid.NewGuid()` results in an exception:

```
System.AggregateException: AggregateException_ctor_DefaultMessage (Arg_CryptographyException)
 ---> System.Security.Cryptography.CryptographicException: Arg_CryptographyException
   at Interop.GetCryptographicallySecureRandomBytes(Byte* , Int32 )
   at System.Guid.NewGuid()
   at Program.Main(String[] args)
   Exception_EndOfInnerExceptionStack
```

Do you have any idea why this happens and how it can be prevented? It works in a normal Blazor app.
It might be related to this [change](https://docs.microsoft.com/en-us/dotnet/api/system.guid.newguid?view=net-6.0#remarks):

> On non-Windows platforms, starting with .NET 6, this function calls the OS's underlying cryptographically secure pseudo-random number generator (CSPRNG) to generate 122 bits of strong entropy. In previous versions of .NET, the entropy is not guaranteed to be generated by a CSPRNG.

**Comment by Elringus** [comment]
Thanks for the info! This should be fixed in #3.

Microsoft provides the implementation only for browsers, hence it wasn't working in vscode (or node): https://github.com/dotnet/runtime/blob/release/6.0/src/libraries/Native/Unix/System.Native/pal_random.js#L27

I've added a [naive polyfill with `Math.random`](https://github.com/Elringus/DotNetJS/blob/77d509f693ba2efad7f5774f27ca5ef0ea0a9b74/Runtime/src/mono.ts#L31), but that's probably not ideal. If someone is aware of a pure JS (without dependencies on node or browser APIs) implementation for crypto-safe random algorithm, please let me know.

**Comment by Skleni** [comment]
Thanks for the fast reply, for our usecase the workaround should be good enough.
